signalstickers_client/classes/downloader.py

```diff
@@ -71,11 +71,15 @@ async def get_sticker(http, sticker_id, pack_id, pack_key):
     """
     Return the content of the webp file for a given sticker
     """
-    sticker_req = await http.get(
+    sticker_resp = await http.get(
         CDN_STICKER_URL.format(pack_id=pack_id, sticker_id=sticker_id),
         timeout=None,
     )
-    sticker_req.raise_for_status()
-    sticker_encrypted = bytes(sticker_req.content)
+    if sticker_resp.status_code == 404:
```

Comment:
Same here, 404 is 403 for Signal ([valid sticker](https://cdn.signal.org/stickers/d9ae4ca540a80a00843660854e6130d2/full/1) - [invalid sticker](https://cdn.signal.org/stickers/d9ae4ca540a80a00843660854e6130d1/full/1))